### PR TITLE
Made PhotoCapabilities.fillLightMode a sequence<FillLightMode>...

### PR DESCRIPTION
**Pull request by miguelao**
This PR:
- Makes `PhotoCapabilities.fillLightMode` a `sequence<FillLightMode>`, 
- removes `unavailable` from `FillLightMode`, since the condition of not-having-flash is encoded in returning an empty sequence and it doesn't make sense to use said `unavailable` for `PhotoSettings`,
- updates some text

### CODE REVIEW

**Comment by jan-ivar** [comment]
@miguelao This seems counter to https://github.com/w3c/mediacapture-image/issues/156, should we resolve that one first?

**Comment by miguelao** [comment]
#156 is independent: in this PR I'm just moving the the idl to match the definitions below. We can talk about `redEyeReduction` separately :-)

File: index.bs [hunk 1]
```diff
@@ -179,10 +179,10 @@ interface ImageCapture {
...
 <pre class="idl">
   interface PhotoCapabilities {
-    readonly attribute FillLightMode      fillLightMode;
-    readonly attribute MediaSettingsRange imageHeight;
-    readonly attribute MediaSettingsRange imageWidth;
-    readonly attribute sequence&lt;boolean>  redEyeReduction;
+    readonly attribute sequence&lt;boolean>       redEyeReduction;
+    readonly attribute MediaSettingsRange      imageHeight;
+    readonly attribute MediaSettingsRange      imageWidth;
+    readonly attribute sequence&lt;FillLightMode> fillLightMode;
```
Comment (jan-ivar) [inline index.bs]: Actually, a sequence [cannot be an attribute](https://heycam.github.io/webidl/#idl-attributes). It's invalid WebIDL. We'd need:
```js
readonly attribute FrozenArray<boolean>    redEyeReduction;
readonly attribute FrozenArray<boolean>    fillLightMode;
```

Comment (miguelao) [inline index.bs]: Changed in the latest commit.

**Pull request merged**